polls/models.py

```diff
@@ -1,3 +1,54 @@
+from tabnanny import verbose
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.contrib.auth import get_user_model
 
-# Create your models here.
+class User(AbstractUser):
+    first_task_of_this_session_performed_at = models.DateTimeField(null=True)
+    is_locked = models.BooleanField(default=False)
+
+class Batch(models.Model):
+    created_at = models.DateTimeField()
+    is_gold = models.BooleanField(default=False)
+    notes = models.TextField()
+
+    class Meta:
+        verbose_name_plural = "batches"
+
+
+class CurrentBatch(models.Model):
```

Comment:
This is wrong. We can't have multiple `CurrentBatch` rows. But this model allows that. Is there another way to hold these global variables in the database? Or can you add appropriate constraints so there is always only one row??